src/odemis/driver/xt_client.py

```diff
@@ -2279,6 +2286,51 @@ def _moveTo(self, future: futures.Future, pos: Dict[str, float], rel: bool = Fal
                 # know the user that the move is not complete.
                 if future._must_stop.is_set():
                     raise CancelledError()
+
+                # The stage is not moving anymore, but we still want to wait until the position has been updated
+                # TODO: base the timeout on the stage movement time estimation (est. *10)
+                timeout = time.time() + 10  # s
+                not_updated = True
+                wait_duration = 20e-03  # s
+
+                # Check for presence of rx/t or rz/r keys in pos
+                # In FM mode, the stage will maintain its rotation and tilt, if it is requested to change
+                # it is a different imaging mode
+                # TODO: extend the behavior to other imaging modes
+                if 'r' in pos or 't' in pos:
+                    logging.debug("Position requested to move includes rx or rz. Reported position accuracy is not"
+                                  "checked.")
```

Comment:
this might get confusing during debugging, since you are checking for `'r'` and `'t'`, but the log reports that `rx` and `rz` are not present. 
```suggestion
                    logging.debug("Position requested to move to includes t (rx or rotation) or r (rz or tilt). Reported position accuracy is not"
                                  "checked.")
```